topi/python/topi/nn/winograd_util.py

```diff
@@ -149,3 +151,16 @@ def winograd_transform_matrices(tile_size, kernel_size, out_dtype):
         const_matrix(B_data.astype(out_dtype), "B"),
         const_matrix(G_data.astype(out_dtype), "G"),
     )
+
+def enum_tile_sizes(data):
+    """Propose tile sizes for winograd convolution"""
+    _, _, H, _ = get_const_tuple(data.shape)
+
+    sizes = [2,]
+    # 2 always present
+    for s in range(3, 9):
+    # overlap less then half
```

Comment:
@merrymercy , @FrozenGene ,
What is your opinion on this approach to construct ```tile_size``` ?

Reply:
I have no strong preference what way should we do. If we think it is a bit trick, we could add comment explaining what happened here, then I think it is enough.

Reply:
I don't have experiences tuning all these tile sizes. 
For H = 56, it will return [3, 4, 5, 6, 7, 8]. It will be good if you can try to prune some candidates that are very likely to be the optimal ones according to your tuning results.

Reply:
@merrymercy ,

*  Inappropriate candidates will be dropped out early by ```xgb```, they don't add overhead or lengthen the search time by much. There can be surprises especially picking e.g. tile_size = 3 sometimes.
* Actual code cover well 2 & 4 cases (tests shows also 3 have good results). But we can be more precise only if narrow down space of ```P``` in accord with ```tile_size``` as you mentioned, or let xgb choose its way.